### ISSUE TITLE
Error: Failed with status '(0x10: MEM_ERROR)' (expected '(0x00: SUCCESS)') 

### ISSUE THREAD

**Issue by Epstone** [ioBroker/ioBroker.zigbee#1292]
Since upgrading from 1.5.6 to 1.6.6 my iobroker seems to need a restart after a few days and is not as stable as before. Zigbee devices are no longer reacting to any remote control actions anymore. The log is full of errors. Maybe you can have a short look. I have not yet upgraded my js host. Maybe my sdcard got corrupt if I am the only one with those issues. It just started with the upgrade, thats weird.

I am just downgrading to 1.5.6 and want to see if the errors stay.

Some exemplary errors or just scroll over the log file:
```
2021-12-11 19:07:20.087  - error:

zigbee.0 (21877) Error 26 on send command to 0x7cb03eaa0a0bd806. Error:
Error: Command 0x7cb03eaa0a0bd806/3 genOnOff.off({},
{"sendWhenActive":false,"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false})
failed (Data request failed with error: 'MAC no resources' (26))
 
at ZStackAdapter.sendZclFrameToEndpointInternal
(/opt/iobroker/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:390:23)
at Queue.executeNext (/opt/iobroker/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)
 
2021-12-11 19:07:00.318 - warn: zigbee.0 (21877) Failed to ping 0x7cb03eaa0a0c1371 Plug 01
host.raspi1 Caught by controller[1]: Error: SREQ '--> ZDO -

extRouteDisc - {"dstAddr":27369,"options":0,"radius":30}' failed with
status '(0x10: MEM_ERROR)' (expected '(0x00: SUCCESS)')

```
[iobroker.2021-12-11.htm.log](https://github.com/ioBroker/ioBroker.zigbee/files/7697636/iobroker.2021-12-11.htm.log)
[iobroker.2021-12-10.log.htm.log](https://github.com/ioBroker/ioBroker.zigbee/files/7697637/iobroker.2021-12-10.log.htm.log)

**Comment by asgothian** [comment]
With that limited amount of information, i will need to use my crystal ball to debug:

You have
- a cc2531 stick
- a network with more than 10 to 15 devices
- a network which runs reasonably fine for x days (x depending on the size of your network and the bandwidth available in the chosen 2.4GHz Band)

This can be seen by this message: `failed with status '(0x10: MEM_ERROR)' (expected '(0x00: SUCCESS)')
`

Your chances to fix this:

- Refresh the cc2531 firmware.
- do a hard reset on the cc2531.
- change coordinator to something more potent.

The TI implementation of the zigbee Stack for the cc2531 has a confirmed bug which will eat memory in specific cases. Due to the CC2531 being outdated hardware, TI has declined to fix this, and it (to my knowledge) outside of the scope accessible to the open source community.

A.
p.s. Older adapter versions may have had the same problem without reporting it.

**Comment by Epstone** [comment]
Thank you very much for analysing my situation and my appologies for the incomplete report. I just bought a recommended CC2652 stick.
